packages/alias-rewrite/src/cli.ts

```
@@ -1,15 +1,16 @@
 #!/usr/bin/env node
 import fg from "fast-glob";
 import { Project, SyntaxKind } from "ts-morph";
-import { mkAliasRewriter, mkRelativeToJs } from "./lib.js";
+console.warn("@promethean/alias-rewrite is deprecated; use @promethean/naming instead.");
+import { mkAliasRewriter, mkRelativeToJs } from "@promethean/naming";
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)